microsoft-edge/dualengine/get-started.md

```diff
@@ -17,7 +17,7 @@ This article walks you through the steps to start using the DualEngine API.
 <!-- ====================================================================== -->
 ## Step 1: Get access to the DualEngine Limited Access Feature
 
-The DualEngine API is part of a Limited Access Feature. For more information or to request an unlock token, contact [Microsoft Support](https://aka.ms/LAFAccessRequests).
+The DualEngine API is part of a Limited Access Feature. For more information or to request an unlock token, contact [Microsoft Support](https://go.microsoft.com/fwlink/?linkid=2271232&clcid=0x409).

```

Comment:
```suggestion
The DualEngine API is part of a Limited Access Feature. For more information or to request an unlock token, fill in the form [Limited Access Feature APIs Request Access Token](https://go.microsoft.com/fwlink/?linkid=2271232&clcid=0x409).
```
The link does not end up at "Microsoft Support" in any discernable sense; the word "support" doesn't appear anywhere in the web page/browser.  
The resulting url is not "support", but office.com.  
Per style guides, link text should match the destination title, which is, per the tab title (= when you paste url into Word), "Limited Access Feature APIs Request Access Token".   Else disorienting nav UX.
Result: 
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/d92205af-18c8-427e-a5ba-4a649127d9fd)